club/style.css

```diff
@@ -37,6 +37,10 @@ GNU General Public License for more details.
 	-webkit-font-smoothing: antialiased;
 }
 
+a {
+	text-underline-offset: .3em;
```

Comment:
offset of `.4rem` or `.15em` matching the designs. Since `em` is relative to parent context, `.15em` is appropriate to use here.

Reply:
done!